db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
```

Comment:
In theory same label on say nodes vs pods might have different meaning, and deserve different treatment.
Let's say I want to tag only container nodes with label region=infra but not other entity types that have that label, so I'll set `labeled_resource_type: "ContainerNode", label_name: "region", label_value: "infra"`.

Admittedly that's a silly example.
In practice, I don't have an example of "region" having a different meaning on other entities,
or of *any* case where constraining by resource_type is important!

[See https://gist.github.com/cben/2619583f777ddb480a10#file-labels-txt for actually occuring labels I've seen, and https://github.com/ManageIQ/manageiq/pull/7605#issuecomment-206920881 for my analysis which labels a user might want to map to tags.]

But since kubernetes/openshift users may invent whatever labels they see fit,
it seemed like something the person defining a mapping _could_ want.

@simon3z Please chime in if you have better idea if per-type mapping is useful.